server/src/main/java/org/opensearch/cluster/LocalNodeResponse.java

```diff
@@ -57,4 +57,11 @@ public int hashCode() {
         return Objects.hash(localNode);
     }
 
+    /**
+     * Get the local node
+     */
+    public DiscoveryNode getLocalNode() {
+        return this.localNode;
+    }
```

Comment:
Can we add a small test to verify this getter value?

Reply:
Sure ill add this in

Reply:
(Style/entirely personal preference) Would really like this further up in the class than below the equals and hashcode, although others prefer getters and setters at the bottom.